modules/minimp3/audio_stream_mp3.cpp

```diff
@@ -232,6 +253,10 @@ bool AudioStreamMP3::is_monophonic() const {
 	return false;
 }
 
+void AudioStreamMP3::get_parameter_list(List<Parameter> *r_parameters) {
+	r_parameters->push_back(Parameter(PropertyInfo(Variant::BOOL, "looping", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_CHECKABLE), Variant()));
```

Comment:
And here. `Variant()` is the default value for the constructor.

Reply:
That is okay because It's a checkable property (if Variant() it means the property is unused).

Reply:
I mean Parameter constructor is defined like this
`Parameter(const PropertyInfo &p_info = PropertyInfo(), const Variant &p_default_value = Variant())`
The `variant()` you use here is part of the constructor and it's unnecessary.